WebRoot/js/ajax/dwt/widgets/DwtCheckbox.js

```diff
@@ -317,13 +336,23 @@ function(templateId, data) {
     data.value = this._initValue;
 	data.type = this.INPUT_TYPE;
 	DwtControl.prototype._createHtmlFromTemplate.call(this, templateId, data);
+	//set custom style for check box only
+	this.CUSTOM_CHECKBOX_STYLE = DwtCheckbox.CUSTOM_CHECKBOX && this.INPUT_TYPE == 'checkbox';
```

Comment:
Done: https://github.com/Zimbra/zm-ajax/pull/4/files#diff-5d22f86ec071d8b2e4f423ae9803c185R111 and https://github.com/Zimbra/zm-ajax/pull/4/files#diff-5d22f86ec071d8b2e4f423ae9803c185R340